ext/curl/curl.stub.php

```diff
@@ -35,6 +35,8 @@ function curl_exec(CurlHandle $handle): string|bool {}
 
 function curl_file_create(string $filename, ?string $mime_type = null, ?string $posted_filename = null): CURLFile {}
 
+function curl_file_create_from_string(string $file_content, string $posted_filename, ?string $mime_type = null): CURLStringFile {}
+
```

Comment:
The `$file_content` parameter name could be shorter. Maybe something like `$content` or `$data` could work. From the context it should be sufficiently clear what the parameter means.

Reply:
Agree, updated to "data"